src/ng/http.js

```diff
@@ -42,6 +42,7 @@ function $HttpParamSerializerProvider() {
       var parts = [];
       forEachSorted(params, function(value, key) {
         if (value === null || isUndefined(value)) return;
+        if (typeof value === 'function') return;
```

Comment:
@carlfranz, you can amend your last commit with `git commit --amend`.
(Or if you want to change/reorganize multiple commits, `git rebase --interactive <COMMIT_RANGE>` is powerful tool. Too powerful - ue at your own risk 😛)